components/brave_wallet/browser/internal/hd_key_unittest.cc

```diff
@@ -581,16 +595,41 @@ TEST(HDKeyUnitTest, GenerateFromV3UTC) {
 
 // https://github.com/bitcoin/bips/blob/master/bip-0173.mediawiki#examples
 TEST(HDKeyUnitTest, GetSegwitAddress) {
-  std::vector<uint8_t> pirvate_key_bytes(32, 0);
-  pirvate_key_bytes.back() = 1;
+  std::vector<uint8_t> private_key_bytes(32, 0);
+  private_key_bytes.back() = 1;
   std::unique_ptr<HDKey> hdkey =
-      HDKey::GenerateFromPrivateKey(pirvate_key_bytes);
+      HDKey::GenerateFromPrivateKey(private_key_bytes);
   EXPECT_EQ(
       base::HexEncode(hdkey->GetPublicKeyBytes()),
       "0279BE667EF9DCBBAC55A06295CE870B07029BFCDB2DCE28D959F2815B16F81798");
-  EXPECT_EQ(hdkey->GetSegwitAddress(),
+  EXPECT_EQ(hdkey->GetSegwitAddress(false),
             "bc1qw508d6qejxtdg4y5r3zarvary0c5xw7kv8f3t4");
-  // TODO(apaymyshev): support testnet and P2WSH.
+  EXPECT_EQ(hdkey->GetSegwitAddress(true),
+            "tb1qw508d6qejxtdg4y5r3zarvary0c5xw7kxpjzsx");
+  // TODO(apaymyshev): support P2WSH.
+}
+
+TEST(HDKeyUnitTest, SignBitcoin) {
```

Comment:
nit: could be useful to add a few more of the cases you link to here so that we have more than just the 1.

Reply:
added a TODO